spec/typescript/exceljs.spec.ts

```diff
@@ -1,40 +1,40 @@
-import 'regenerator-runtime/runtime';
+import "regenerator-runtime/runtime";
 
-import { expect } from 'chai';
-import ExcelJS from '../../index';
+import { expect } from "chai";
+import * as ExcelJS from "../../index";

```

Comment:
That's actually breaking compatibility change, I don't see reason of doing that now. Also importing everything from exceljs works great for my projects. 
What kind of error have you got?

Reply:
I would be glad for sharing me your test repo from screenshots.

Reply:
I just pushed the repo here: https://github.com/groozin/exceljs-test 
That being said - using the `vscode` version 

```
Version: 1.85.1 (Universal)
Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
Date: 2023-12-13T09:48:06.308Z
Electron: 25.9.7
ElectronBuildId: 25551756
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin x64 23.1.0
```

and `node 18` I cant reproduce anymore. Still using the version `4.30` of the `exceljs`. 
Maybe the recent `vscode` versions fixed the problem. 
But just in case I uploaded the repo I used to test.